components/Advocates/AdvocatesQampProjects.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+  <section class="qamp-projects">
+    <UiFiltersResultsLayout class="qamp-projects__filters-result-section">
+      <template #filters-on-m-l-screen>
+        <UiFieldset :label="filter.label">
+          <client-only>
+            <bx-checkbox
+              v-for="option in filter.options"
+              :key="option"
+              :checked="isCohortFilterChecked(option)"
+              :label-text="option"
+              :value="option"
+              @bx-checkbox-changed="
+                updateCohortFilter(option, $event.target.checked)
+              "
+            />
+          </client-only>
+        </UiFieldset>
+      </template>
+      <template #filters-on-s-screen>
+        <UiMultiSelect
+          :label="filter.label"
+          :options="filter.options"
+          :value="cohortFiltersAsString"
+          @change-selection="updateCohortFilters($event)"
+        />
+      </template>
+      <template #results>
+        <div class="cds--row">
+          <div
+            v-for="project in filteredProjects"
+            :key="project.name"
+            class="cds--col-max-8 qamp-projects__card"
+          >
+            <UiCard
+              :title="project.name"
+              :tags="[project.cohort.name]"
+              cta-label="Go to repo"
+              :segment="{
+                cta: project.githubIssueUrl,
```

Comment:
Hoping that this data will be updated... so that if we say "Go to repo", users are actually taken to a repo, and not an `api.github` link 


https://github.com/Qiskit/qiskit.org/assets/6276074/095bf694-bf55-4bb2-9da9-388bb2160a7f